src/components/InventoryList.tsx

```diff
@@ -52,6 +53,11 @@ const InventoryList: React.FC = () => {
     }
   };
 
+  const addToBookmarksHandler = (event: any) => {
```

Comment:
Maybe one day I will understand WTF typescript wants from me, but it's not today. Or @mmaelicke you can give me a hint. I am always struggling with this situation. :(

Reply:
If you are refering to the use of `any` here, you need to add the correct type. Typescript can't tell automatically, because the link to the calling context is not (yet) clear here

Reply:
Yes, I use `any` because I was super frustrated after 20 minutes of googling and trial and error...
Do you have a working strategy to add types you don't know?

Reply:
Hover over the `e` in the components `onClick` attribute and VSCode should show you some type hints.